Branch-SDK/Branch-SDK/Branch.m

```diff
@@ -384,6 +384,9 @@ - (BOOL)handleDeepLink:(NSURL *)url {
 }
 
 - (BOOL)continueUserActivity:(NSUserActivity *)userActivity {
+    //always save the incoming url in the preferenceHelper, as the externalIntentURI
+    self.preferenceHelper.externalIntentURI = [userActivity.webpageURL absoluteString];
```

Comment:
Note that this is saving (and later sending, in OpenRequest) *ALL* incoming urls, including those in no way related to Branch. Not that it's invalid, just pointing it out for your consideration.